src/interface.rs

```diff
@@ -14,7 +14,7 @@ pub struct NetworkInterface {
     /// Interface's name
     pub name: String,
     /// Interface's address
-    pub addr: Option<Addr>,
+    pub addr: Vec<Addr>,
```

Comment:
I really like this! This was one of the first bad impressions after I published the first version of the crate, Im glad you addressed it!